### PR TITLE
updating TimeSpan.xml for Try .NET

### PR DESCRIPTION
**Pull request by WilliamAntonRohm**
per https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1558837/

### CODE REVIEW

**Comment by WilliamAntonRohm** [comment]
samples PR is https://github.com/dotnet/samples/pull/1453

**Comment by mairaw** [comment]
I'm rebuilding the master branch right now in order to be able to pull the latest samples changes. @BillWagner can you close and reopen this PR in the morning so it's ready for me to review when I start?

File: xml/System/TimeSpan.xml [hunk 1]
```diff
@@ -69,34 +69,38 @@
...
 -   By calling a method or performing an operation that returns a <xref:System.TimeSpan> value. For example, you can instantiate a <xref:System.TimeSpan> value that represents the interval between two date and time values, as the following example shows.  
   
-     [!code-csharp[System.TimeSpan.Class#4](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.timespan.class/cs/instantiate1.cs#4)]
+     [!code-csharp-interactive[System.TimeSpan.Class#4](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.timespan.class/cs/instantiate1.cs#4)]
      [!code-vb[System.TimeSpan.Class#4](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.timespan.class/vb/instantiate1.vb#4)]  
   
      You can also initialize a <xref:System.TimeSpan> object to a zero time value in this way, as the following example shows.  
   
-     [!code-csharp[System.TimeSpan.Class#6](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.timespan.class/cs/zero1.cs#6)]
+     [!code-csharp-interactive[System.TimeSpan.Class#6](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.timespan.class/cs/zero1.cs#6)]
```
[the remark this answers is not in the document]
Comment (WilliamAntonRohm) [reply xml/System/TimeSpan.xml]: @mairaw, @BillWagner -- there are quite a few such instances of helper functions that could be put into main() as local functions -- please let me know if should go after them

[the remark this answers is not in the document]
Comment (mairaw) [reply xml/System/TimeSpan.xml]: I'd wait for @BillWagner's opinion

[the remark this answers is not in the document]
Comment (BillWagner) [reply xml/System/TimeSpan.xml]: @WilliamAntonRohm I would prefer updating the helper functions to local functions so we can add the interactivity.  That would be fantastic.

[the remark this answers is not in the document]
Comment (WilliamAntonRohm) [reply xml/System/TimeSpan.xml]: @mairaw, @BillWagner -- thank you for the go-ahead.  In this API, there are 3 samples with a total of 6 private functions.  There are a lot more in other APIs -- I'll let you know those numbers as I go back through the APIs we've done.

[the remark this answers is not in the document]
Comment (WilliamAntonRohm) [reply xml/System/TimeSpan.xml]: Updated two of the local-function samples in PR https://github.com/dotnet/samples/pull/1453, now merged, and updated those two lines in TimeSpan.xml

**Comment by WilliamAntonRohm** [comment]
#sign-off

**Comment by mairaw** [comment]
Need a fresh build

**Pull request merged**